apps_product_creator/models/product_product.py

```diff
@@ -2,6 +2,28 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 from odoo import api, fields, models
 from odoo import tools
+import lxml
+
+
+def urljoin(*args):
+    """
+    Joins given arguments into an url. Trailing but not leading slashes are
+    stripped for each argument.
+    """
+
+    return "/".join(map(lambda x: str(x).rstrip('/'), args))
+
+
+def hook_github_image_url(rst_desc, github_url):
```

Comment:
why `hook_`?

Reply:
it is literally processing the URL and adding GitHub module prefix URL for display purpose.